src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -0,0 +1,257 @@
+        var queryParameters: SurveyAnswersQuery = {
+            surveyName 
+        };
+
+        queryParameters.stepIdentifier = stepId;
+        queryParameters.resultIdentifier = resultId;
+        
+        if (pageID) {
+            queryParameters.pageID = pageID;
+        }
+
+        return MyDataHelps.querySurveyAnswers(queryParameters);
+    }
+
+    function initializeWeekPager() : Date {
+        const today = startOfDay(new Date());
+        const monday : Date = isMonday(today) ? today : previousMonday(today);
+        const sunday : Date = nextSunday(monday);
+        return monday;
+    }
+
+    function pageWeek(newStart: Date) {
+        pageWeeklyData( newStart );
+    }
+
+    function pageWeeklyData(startOfWeek: Date, freshData? : IBloodPressureDumbbell[]){
+        const useData = freshData ?? bpData;
+        const weekData : Dumbbell[] = [];
+        for (let i = 0; i < 7; i++) {
+            var currentDate = startOfDay(addDays(startOfWeek, i));
+            var dataForDay = useData.find( db => isEqual(db.date, currentDate));
+            if (!dataForDay)
+            {
+                var formattedDate = format(currentDate, "MM/dd");
+                weekData.push({xValue : formattedDate});
+            } 
+            else 
+            {
+                weekData.push(dataForDay.dumbbell);
+            }
+        }
+
+        setWeeklyDataForViz(weekData);
+        setStartOfWeek(startOfWeek);
+    }
+
+    function buildInterval(entries : number[]){
+        const min = Math.min(...entries);
+        const closedInterval : ClosedInterval = {values : [min]};
+        
+        var max = min;
+        if (entries.length > 1){
+            max = Math.max(...entries);
+        }
+        closedInterval.values.push(max);
+        return closedInterval;
+    }
+
+    function assignClass(avgSystolic : number, avgDiastolic : number){
+
+        if ((avgSystolic >=90 && avgSystolic <= 119) && (avgDiastolic >= 60 && avgDiastolic <= 79)){
+            return DumbbellClass.inRange;
+        }
+
+        return DumbbellClass.outOfRange;
+    }
+
+    useEffect(() => {
+        let debouncedInitialize = debounce(initialize, 500);
+ 
+        debouncedInitialize();
+        mdh.on("applicationDidBecomeVisible", debouncedInitialize);
+         
+        return () => {
+            mdh.off("applicationDidBecomeVisible", debouncedInitialize);
+        }
+     }, []);
+
+    if (loadingData) 
+    {
+        return <LoadingIndicator/>;
+    }
+    else
+    {
+        return (
+            <div className="bloodPressureVisualization">
+                <div className="bpTitle">Blood Pressure</div>
+                <div>
+                    <div className="pagerRow">
+                        <DateRangeNavigator intervalType="Week" intervalStart={startOfWeek} onIntervalChange={pageWeek}></DateRangeNavigator>
```

Comment:
There is a border bottom on this that looks like a yaxis line, Any way to remove this?

Reply:
This actually seems correct to have a bottom border?

<img width="467" alt="Screenshot 2023-11-14 at 10 23 05 AM" src="https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/a0193ce5-606a-4145-bfbc-cb80ad09063d">

That being said, the DateRangeNavigator should potentially have another variant if you want to achieve the look/feel you want here with the different blue buttons for left/right